shared/profile/edit-profile/index.js

```diff
@@ -35,6 +35,7 @@ export default compose(
     bioLengthLeft: props.bio ? maxProfileBioChars - props.bio.length : maxProfileBioChars,
   })),
   withHandlers({
+    onCancel: ({onBack}) => () => onBack(),
```

Comment:
Hm, not sure about that -- I need both an onBack (for the back button) and an onCancel (for the cancel button) and renameProp appears to be destructive of the source prop.  (Just tried it.)